test/Test/Main.purs

```diff
@@ -288,3 +288,201 @@ numbersTestCode = do
+  acos (cos pi_2) `assertNearlyEqual` pi_2
+
+  log "Data.Number.atan"
+  log "  atan (tan 0.0) = 0.0"
+  atan (tan 0.0) `assertNearlyEqual` 0.0
+
+  log "  atan (tan pi / 4.0) = pi / 4.0"
+  atan (tan pi_4) `assertNearlyEqual` pi_4
+
+  log "  atan (tan pi / 2.0) = pi / 2.0"
+  atan (tan pi_2) `assertNearlyEqual` pi_2
+
+  log "Data.Number.atan2"
+  log "  atan2 1.0 2.0 = atan (1.0 / 2.0)"
+  atan2 1.0 2.0 `assertNearlyEqual` atan (1.0 / 2.0)
+
+  log "Data.Number.log"
+  log "  log 2.0 = ln2"
+  Num.log 2.0 `assertNearlyEqual` ln2
+
+  log "  log 10.0 = ln10"
+  Num.log 10.0 `assertNearlyEqual` ln10
+
+  log "Data.Number.exp"
+  log "  exp ln2 = 2.0"
+  exp ln2 `assertNearlyEqual` 2.0
+
+  log "  exp ln10 = 10.0"
+  exp ln10 `assertNearlyEqual` 10.0
+
+  log "Data.Number.abs"
+  log "  abs pi = pi"
+  assert $ abs pi == pi
+
+  log "  abs (-pi) = pi"
+  assert $ abs neg_pi == pi
+
+  log "Data.Number.sign"
+  log "  sign pi = 1.0"
+  assert $ sign pi == 1.0
+
+  log "  sign (-pi) = -1.0"
+  assert $ sign neg_pi == -1.0
+
+  log "Data.Number.sqrt"
+  log "  sqrt 2.0 = sqrt2"
+  sqrt 2.0 `assertNearlyEqual` sqrt2
+
+  log "  sqrt (1.0 / 2.0) = sqrt1_2"
+  sqrt (1.0 / 2.0) `assertNearlyEqual` sqrt1_2
+
+  log "Data.Number.pow"
+  log "  2.0 `pow` (1.0 / 2.0) = sqrt2"
+  (2.0 `pow` (1.0 / 2.0)) `assertNearlyEqual` sqrt2
+
+  log "Data.Number.min"
+  log "  min 0.0 1.0 = 0.0"
+  assert $ min 0.0 1.0 == 0.0
+
+  log "Data.Number.max"
+  log "  max 0.0 1.0 = 1.0"
+  assert $ max 0.0 1.0 == 1.0
+
+  log "Data.Number rounding"
+  log "  ceil 4.7 = 5.0"
+  assert $ ceil 4.7 == 5.0
+
+  log "  floor 4.7 = 4.0"
+  assert $ floor 4.7 == 4.0
+
+  log "  round 4.7 = 5.0"
+  assert $ round 4.7 == 5.0
+
+  log "  trunc 4.7 = 4.0"
+  assert $ trunc 4.7 == 4.0
+
+  log "  ceil (-4.7) = -4.0"
+  assert $ ceil (-4.7) == -4.0
+
+  log "  floor (-4.7) = -5.0"
+  assert $ floor (-4.7) == -5.0
+
+  log "  round (-4.7) = -5.0"
+  assert $ round (-4.7) == -5.0
+
+  log "  trunc (-4.7) = -4.0"
+  assert $ trunc (-4.7) == -4.0
+
+  log "  ceil 4.3 = 5.0"
+  assert $ ceil 4.3 == 5.0
+
+  log "  floor 4.3 = 4.0"
+  assert $ floor 4.3 == 4.0
+
+  log "  round 4.3 = 4.0"
+  assert $ round 4.3 == 4.0
+
+  log "  trunc 4.3 = 4.0"
+  assert $ trunc 4.3 == 4.0
+
+  log "  ceil (-4.3) = -4.0"
+  assert $ ceil (-4.3) == -4.0
+
+  log "  floor (-4.3) = -5.0"
+  assert $ floor (-4.3) == -5.0
+
+  log "  round (-4.3) = -4.0"
+  assert $ round (-4.3) == -4.0
+
+  log "  trunc (-4.3) = -4.0"
+  assert $ trunc (-4.3) == -4.0
+
+  log "Data.Number.remainder (%)"
+  log "  12.0 % 5.0 = 2.0"
+  assert $ 12.0 % 5.0 == 2.0
+
+  log "  (-12.0) % 5.0 = 2.0"
+  assert $ (-12.0) % 5.0 == -2.0
+
+  log "Data.Number constants"
+  log "  log10e = 1.0 / ln10"
+  log10e `assertNearlyEqual` (1.0 / ln10)
+
+  log "  log2e = 1.0 / ln2"
+  log2e `assertNearlyEqual` (1.0 / ln2)
+
+assertNearlyEqual :: Number -> Number -> Effect Unit
+assertNearlyEqual x y = assert ((abs (x - y)) < 1e-12)
```

Comment:
This probably ought to use the function from Data.Number.Approximate?

Reply:
I didn't see eqAbsolute down the bottom there. Fixed in 91c9a2e18a6e90c6e3d91142e5777deba5da23df